.github/workflows/lfric_test.yml

```diff
@@ -112,26 +115,30 @@ jobs:
         spack load lfric-build-environment%gcc@${GCC_VERSION}
         source .runner_venv/bin/activate
         export PSYCLONE_LFRIC_DIR=${GITHUB_WORKSPACE}/examples/lfric/scripts
-        export PSYCLONE_CONFIG_FILE=${PSYCLONE_LFRIC_DIR}/KGOs/lfric_psyclone_no_annexed.cfg
+        # The no-annex-dof configuration fails in LFRic because some kernels lie in their metadata
```

Comment:
I know this isn't strictly our issue but possibly we need one to track this as it's important. Alternatively, we could point to an MO ticket if there is one.